tests/src/test/scala/templates/ReminderSlackTests.scala

```diff
@@ -21,111 +21,488 @@ package packages
 import common.{TestHelpers, Wsk, WskProps, WskTestHelpers}
 import java.io._
-// import spray.json.DefaultJsonProtocol.StringJsonFormat
-// import spray.json.pimpAny
+import com.jayway.restassured.RestAssured
+import com.jayway.restassured.config.SSLConfig
+import spray.json.DefaultJsonProtocol._
+import spray.json._
+import scala.concurrent.duration.DurationInt
+import scala.language.postfixOps
 
 @RunWith(classOf[JUnitRunner])
 class ReminderSlackTests extends TestHelpers
-    with WskTestHelpers
-    with BeforeAndAfterAll {
-
-    implicit val wskprops = WskProps()
-    val wsk = new Wsk()
-
-    //set parameters for deploy tests
-    val nodejs6folder = "../runtimes/nodejs-6/actions";
-    val nodejs8folder = "../runtimes/nodejs/actions";
-    val phpfolder = "../runtimes/php/actions";
-    val pythonfolder = "../runtimes/python/actions";
-    val swiftfolder = "../runtimes/swift/actions";
-
-    behavior of "Get Slack Reminder Template"
-
-    /**
-     * Test the nodejs 6 "Get Slack Reminder Template" template
-     */
-     it should "invoke nodejs 6 send-message.js and get the result" in withAssetCleaner(wskprops) { (wp, assetHelper) =>
-       val name = "messageNode"
-       val file = Some(new File(nodejs6folder, "send-message.js").toString());
-       assetHelper.withCleaner(wsk.action, name) { (action, _) =>
-         action.create(name, file)
-       }
-
-       withActivation(wsk.activation, wsk.action.invoke(name)) {
-         activation =>
-          activation.response.success shouldBe true
-          activation.response.result.get.toString should include("Your scrum is starting now.  Time to find your team!")
-       }
-     }
-     /**
-      * Test the nodejs 8 "Get Slack Reminder Template" template
-      */
-      it should "invoke nodejs 8 send-message.js and get the result" in withAssetCleaner(wskprops) { (wp, assetHelper) =>
-        val name = "messageNode"
-        val file = Some(new File(nodejs8folder, "send-message.js").toString());
-        val kind = Option("nodejs:8")
-
-        assetHelper.withCleaner(wsk.action, name) { (action, _) =>
-          action.create(name, file, kind)
-        }
-
-        withActivation(wsk.activation, wsk.action.invoke(name)) {
-          activation =>
-           activation.response.success shouldBe true
-           activation.response.result.get.toString should include("Your scrum is starting now.  Time to find your team!")
-        }
+  with WskTestHelpers
+  with BeforeAndAfterAll {
+
+  implicit val wskprops = WskProps()
+  val wsk = new Wsk()
+  val allowedActionDuration = 120 seconds
+
+  // statuses for deployWeb
+  val successStatus =
+    """"status":"success""""
+
+  val deployTestRepo = "https://github.com/ibm-functions/template-reminder-slack"
+  val slackReminderAction = "myPackage/send-message"
+  val slackSequence = "myPackage/post_message_slack_sequence"
+  val packageName = "myPackage"
+  val triggerName = "myTrigger"
+  val ruleName = "myRule"
```

Comment:
you should make these asset names unique (such as appending timestamp) in case cleanup fails.

Reply:
Okay, did this here as well as in the other template repos.